examples/advanced_pytorch/server.py

```diff
@@ -99,7 +99,7 @@ def main():
         fraction_evaluate=0.2,
         min_fit_clients=2,
         min_evaluate_clients=2,
-        min_available_clients=10,
+        min_available_clients=2,
```

Comment:
If we default to `--toy True`, can we continue to run it with 10 clients?

Reply:
Ok so it turns out this happens even with 2 clients, so there must be another issue at hand.

Reply:
@charlesbvll why are we not waiting until all 10 clients are available?

Reply:
Oh right, this is something I forgot to revert, good catch!